src/main/webapp/CreateProfilePage.html

```diff
@@ -59,16 +65,15 @@ <h1 class="text-center">Create Your Profile Page</h1>
         </div>
       </form>
     </div>
-
     <script>
```

Comment:
I decided to keep the script in the html file because it works better this way.

Reply:
Can you elaborate a little? Ideally all javascript are in script files and only html should be in html files. Are you running into issues?

Reply:
When I separated the script into a separate file, the validation didn't work the way I wanted it to. It wasn't changing the color of the form.

Reply:
Hm, weird, I don't see any reason why it wouldn't work based on the code here. I think it's ok to submit for now, after it's deployed I can debug a little.

(The other benefit of doing this in a js file is that you avoid the code duplication in both html files)

Reply:
+1 what Rodrigo said

Reply:
When you tried ```$(document).ready()``` it didn't fix the problem?

Reply:
Doing $(document).ready() did not work

Reply:
As Rodrigo suggested, lets submit this PR for now and we can debug the issue after.